publish/stack-nodejs-express/stack-nodejs-express.md

```diff
@@ -59,7 +59,6 @@ Kubernetes through a modern DevOps toolchain that is triggered in Git.
 
 - [Docker](https://docs.docker.com/get-started/) must be installed.
 - [Appsody](https://appsody.dev/docs/getting-started/installation) must be installed.
-- *Optional:* If your organisation has customized application stacks, you need the URL that points to an `index.yaml` configuration  file.
 - *Optional*: If you are testing multiple microservices together, you must have access to a Kubernetes cluster for development. For local cluster development on Docker Desktop, you can enable Kubernetes from the menu by selecting *Preferences* -> *Kubernetes* -> *Enable Kubernetes*. Other options include [Minishift](https://www.okd.io/minishift/) or [Minikube](https://kubernetes.io/docs/setup/learning-environment/minikube/). If you want to use remote cluster development, use Codewind.
```

Comment:
- *Optional*: For local cluster development on Docker Desktop, you can enable Kubernetes from the menu by selecting *Preferences* -> *Kubernetes* -> *Enable Kubernetes*. Other options include [Minishift](https://www.okd.io/minishift/) or [Minikube](https://kubernetes.io/docs/setup/learning-environment/minikube/). If you want to use remote cluster development, use Codewind.

Reply:
I suggested removing the entire sentence "If you are testing multiple microservices together, you must have access to a Kubernetes cluster for development."

Using Kubernetes is already stated in the purpose of the guide. It is confusing to condition its usage at this point in the tutorial, because you *must* use Kubernetes to achieve the guide purposes whether or not you are testing multiple microservices.

Reply:
This optional bullet was requested by Alan Little. I believe the rationale is that many users might be following the guide simply to learn how to develop locally.

Reply:
How about we re-organize slightly to provide the most useful piece of information for our local developer. I think the rest of it is already covered in the "Test" section: 

- [Docker](https://docs.docker.com/get-started/) must be installed. For local cluster development on Docker Desktop, you can enable Kubernetes from the menu by selecting *Preferences* -> *Kubernetes* -> *Enable Kubernetes*. Other options include [Minishift](https://www.okd.io/minishift/) or [Minikube](https://kubernetes.io/docs/setup/learning-environment/minikube/).
- [Appsody](https://appsody.dev/docs/getting-started/installation) must be installed.

Reply:
Changes applied. If you are happy please can you approve a review for merging